packages/@aws-cdk/aws-eks/lib/manifest-resource/yaml/composer.py

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
Vendoring this library wholesale is kinda scary to me. What about updates? Shouldn't we package it at build time?

Reply:
Removed. no need!